.vib/vib-platform-verify.json

```diff
@@ -61,8 +61,22 @@
               "endpoint": "lb-wordpress-https",
               "app_protocol": "HTTPS",
               "env": {
-                "username": "user",
-                "password": "S39BKWjSkh"
+                "username": "test_user",
+                "password": "ComplicatedPassword123!4"
+              }
+            }
+          }
+        },
+        {
+          "action_id": "jmeter",
+          "params": {
+            "config": {
+              "test_plan_file": "/.vib/wordpress/wordpress.jmx",
```

Comment:
This is the error from the verification tests:

>/.vib/wordpress/wordpress.jmx (No such file or directory)

The correct path is '.vib/wordpress/jmeter/wordpress.jmx'